sql/core/src/test/resources/sql-tests/results/group-analytics.sql.out

```diff
@@ -223,12 +223,19 @@ grouping_id() can only be used with GroupingSets/Cube/Rollup;
 
 
 -- !query 16
-SELECT course, year, grouping__id FROM courseSales GROUP BY CUBE(course, year)
+SELECT course, year, grouping__id FROM courseSales GROUP BY CUBE(course, year) ORDER BY grouping__id, course, year
```

Comment:
Why you only commit `group-analytics.sql.out`? You should also commit modified `group-analytics.sql`.

Reply:
thanks for your tips